### ISSUE TITLE
Accessibility Issues in the GUI

### ISSUE THREAD

**Issue by Kazaf123** [Nightonke/Saver#58]
Description:
This issue aims to figure out some accessibility issues in the following GUI, and we appreciate if such issues could be fixed, which would bring great assistance for visually impaired users.
 
![image](https://user-images.githubusercontent.com/44084361/160743703-cf1c265f-cfac-4f17-920b-3032d44f59a5.png)

Accessibility Issues:
1.	The menu and App names in the top navigation bar lack content-description, resulting TalkBack or Voice Over could speak these components.
2.	The text of “请输入密码查询您的账本”suffers from the small size issue.
3.	All numeric keys at the bottom lack content-descriptions to support to read.
Appreciate:
Thanks for reading this bug report and I really hope you will repair these issues for people using this App conveniently.

**Comment by HFESimon** [comment]
您好，邮件已收到！
